src/main/java/io/zeebe/monitor/config/KafkaConfig.java

```diff
@@ -0,0 +1,116 @@
+package io.zeebe.monitor.config;
+
+import com.sun.istack.NotNull;
+import org.apache.kafka.common.serialization.Serdes;
+import org.apache.kafka.streams.StreamsConfig;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.kafka.annotation.EnableKafka;
+import org.springframework.kafka.annotation.EnableKafkaStreams;
+import org.springframework.kafka.annotation.KafkaStreamsDefaultConfiguration;
+import org.springframework.kafka.config.KafkaStreamsConfiguration;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Configuration
+@EnableKafka
+@EnableKafkaStreams
+public class KafkaConfig {
```

Comment:
We aim to use spring cloud streams everywhere across vnoc. Please switch to it.
You can check the config and implementation from vnoc-process-manager for example

Reply:
I think so far we only use it in `vnoc-process-manager` while rest of vnoc is using Kafka Streams, I think we should have a discussion with @xulfus on the preferred library

Reply:
At least network-actions-gateway and sponde-services are also using it. 
Not to mention that end result looks much cleaner and readable spring cloud config

Reply:
Unfortunately we are missing documented guidelines on library choices; my bad, and an action point for me. So, we don't really have an "official" and communicated stand on this. As to spring cloud streams, that is probably a good recommendation - but for this particular case we might decide otherwise as we might want to contribute upstream.

TL;DR for now, I believe Kafka streams is okay for this project as the implementation is already done and there were no other guidelines available